app/controllers/challenges_controller.rb

```diff
@@ -24,6 +24,9 @@ def start
   private
   def use_challenge_id!
     @challenge_id = params[:id]
+
+    render_404 if @challenge_id == '12in23'
+
```

Comment:
Could you remove this one please. I'm happy for people to still see their old progress pages. The rest looks great :)